magda-scss-compiler/setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+cd /usr/src/app
+# delete node-sass comes with pancake-sass 2.1.0 T_T
+rm -Rf node_modules/@gov.au/pancake-sass/node_modules/node-sass
+echo "Rebuild node-sass"
+npm install node-sass@4.9.3 stringstream@1.0.0
```

Comment:
@AlexGilleran 
Good idea~ The node-sass reinstallation is only for pulling Linux version binary to docker.
`stringstream` put there just for some wired version issue.
I think, instead of reinstalling this two, I might try something else (that doesn't require specifying a version) to see if it works.